frontend/src/app/actions/browser.js

```diff
@@ -6,13 +6,9 @@ import type {
   SetStateAction
 } from '../reducers/browser';
 
-function setState(payload: BrowserState): SetStateAction {
+export function setState(payload: BrowserState): SetStateAction {
```

Comment:
Using `export default { setState }` as down below prevents using `import { setState } from 'module'`. It just comes back as undefined. It could be used as `import foo from 'module'; foo.setState`, but the {} form is nice to be able to use.